pkgs/development/libraries/science/math/cudnn/generic.nix

```diff
@@ -8,6 +8,14 @@
 , cudatoolkit
 , fetchurl
 , addOpenGLRunpath
+, # The distributed version of CUDNN includes both dynamically liked .so files,
+  # as well as statically linked .a files.  However, CUDNN is quite large
+  # (multiple gigabytes), so you can save some space in your nix store by
+  # removing the statically linked libraries if you are not using them.
+  #
+  # Setting this to true removes the statically linked .a files.
+  # Setting this to false keeps these statically linked .a files.
+  removeStatic ? false
```

Comment:
Hmm, this seems incorrect?  Why would we want to remove static libraries if our `hostPlatform` is statically linked?

Even if you meant:

```suggestion
  removeStatic ? !stdenv.hostPlatform.isStatic
```

then this could potentially still cause breakage for downstream users.  I think it is out of scope for this PR.